specs/namespace/creation-for-whitelist.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the humbug/php-scoper package.
+ *
+ * Copyright (c) 2017 Théo FIDRY <theo.fidry@gmail.com>,
+ *                    Pádraic Brady <padraic.brady@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+return [
+    'meta' => [
+        'title' => 'Namespace declaration creation for whitelisted classes',
+        // Default values. If not specified will be the one used
+        'prefix' => 'Humbug',
+        'whitelist' => [
+        ],
+    ],
+
+    [
+        'spec' => <<<'SPEC'
```

Comment:
good use of the format:

```php
[
    'spec' => '',
    'payload' => '',
]
```

but I think it's a case simple enough where we can use the simplified format:

```php
'spec content' => 'payload content'
```

As you can find [here](https://github.com/humbug/php-scoper/blob/master/specs/misc.php#L34)